spec/QueryTools.spec.js

```diff
@@ -255,6 +255,71 @@ describe('matchesQuery', function () {
     expect(matchesQuery(img, q)).toBe(false);
   });
 
+  it('matches on queries with new format #parse-SDK-JS/pull/1373', function () {
```

Comment:
How is this related to "add support for $eq in LiveQuery"? Isn't this a test for a different PR?

Reply:
Since in https://github.com/parse-community/Parse-SDK-JS/pull/1373, we are changing the format of `query`. There are certain LiveQuery tests(parse-sdk-js) which are failing because parse-server doesn't support new query format for LiveQuery.

So, we added support for new `query` format in LiveQuery in parse-server.

Reply:
Not sure I follow; is https://github.com/parse-community/Parse-SDK-JS/pull/1373 a requirement for this PR?

Reply:
https://github.com/parse-community/Parse-SDK-JS/pull/1373#issuecomment-934451402 please see this.

Reply:
So I understand that the tests in https://github.com/parse-community/Parse-SDK-JS/pull/1373 will only pass when this PR (#7607) has been merged. But they relate to 2 different features:
- add support for $eq in LiveQuery
- add support to "combine equalTo clauses with [other] clauses"

So why is https://github.com/parse-community/Parse-SDK-JS/pull/1373 mentioned in this test case?

Reply:
@mtrezza sorry, I could understand you properly, could you please elaborate a bit more?

Reply:
Sure, it seems these are 2 different features: 

- https://github.com/parse-community/parse-server/pull/7607
- https://github.com/parse-community/Parse-SDK-JS/pull/1373

Even if https://github.com/parse-community/Parse-SDK-JS/pull/1373 never gets merged, we would still want to merge https://github.com/parse-community/parse-server/pull/7607 as a feature, right? So I don't understand why the tests in https://github.com/parse-community/parse-server/pull/7607 mention https://github.com/parse-community/Parse-SDK-JS/pull/1373:
```
it('matches on queries with new format #parse-SDK-JS/pull/1373', function () {
```